presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -401,6 +401,9 @@ protected Void visitAllColumns(AllColumns node, Integer context)
         protected Void visitTable(Table node, Integer indent)
         {
             builder.append(formatName(node.getName()));
```

Comment:
```suggestion
            builder.append(formatName(node.getName())).append(" ");
```

There should exist a space between table name and table version expression.